Webshop/frontend/src/app/components/auth/register/register.component.ts

```diff
@@ -13,6 +13,10 @@ export class RegisterComponent implements OnInit {
   password: string = '';
   confirmPassword: string = '';
 
+  emailPattern = "^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$";
```

Comment:
ha ilyet is ellenőriz afrontend akkor legyen ez: ^(([^<>()\\[\\]\\\\.,;:\\s@\"]+(\\.[^<>()\\[\\]\\\\.,;:\\s@\"]+)*)|(\".+\"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$
mert ez van a backenden is

Reply:
Lecseréltem